openstack/sap-seeds/templates/flavor-seed.yaml

```diff
@@ -430,6 +434,7 @@ spec:
     is_public: true
     extra_specs:
       {{- tuple . "vmware" | include "sap_seeds.helpers.extra_specs" | indent 6 }}
+      "hw:cpu_cores": "48"  # used in nova-vmware as cores-per-socket (24pCPU = 48vCPU)
```

Comment:
We also have older hardware with only 44 cores per socket still in use.

Reply:
I did some more testing and the underlying socket-size is mostly irrelevant. Details in the commit message.

I also fixed some invalid cpus-per-socket values to reflect the new findings.